contracts/interchain-token-service/src/contract.rs

```diff
@@ -260,15 +260,15 @@ impl AxelarExecutableInterface for InterchainTokenService {
     ) {
         let _ = Self::validate_message(&env, &source_chain, &message_id, &source_address, &payload);
 
-        let _ = Self::execute_message(
+        event::executed(
```

Comment:
actually, this isn't needed. the gateway `validate_message` already emits the event, which is meant to be consistent with EVM gateway
https://github.com/axelarnetwork/axelar-cgp-soroban/blob/60b818d31aa250f1d57e2ef259780a2f3d2ed40e/contracts/axelar-gateway/src/contract.rs#L123

there's no need for the app to emit this event (EVM ITS doesn't do it as well)